x-pack/plugins/siem/cypress/integration/cases.spec.ts

```diff
@@ -0,0 +1,115 @@
+  ALL_CASES_OPENED_ON,
+  ALL_CASES_PAGE_TITLE,
+  ALL_CASES_REPORTER,
+  ALL_CASES_REPORTERS_COUNT,
+  ALL_CASES_SERVICE_NOW_INCIDENT,
+  ALL_CASES_TAGS,
+  ALL_CASES_TAGS_COUNT,
+} from '../screens/all_cases';
+import {
+  ACTION,
+  CASE_DETAILS_DESCRIPTION,
+  CASE_DETAILS_PAGE_TITLE,
+  CASE_DETAILS_PUSH_AS_SERVICE_NOW_BTN,
+  CASE_DETAILS_STATUS,
+  CASE_DETAILS_TAGS,
+  CASE_DETAILS_TIMELINE_MARKDOWN,
+  CASE_DETAILS_USER_ACTION,
+  CASE_DETAILS_USERNAMES,
+  PARTICIPANTS,
+  REPORTER,
+  USER,
+} from '../screens/case_details';
+import { TIMELINE_DESCRIPTION, TIMELINE_QUERY, TIMELINE_TITLE } from '../screens/timeline';
+
+import { goToCaseDetails, goToCreateNewCase } from '../tasks/all_cases';
+import { openCaseTimeline } from '../tasks/case_details';
+import { backToCases, createNewCase } from '../tasks/create_new_case';
+import { loginAndWaitForPageWithoutDateRange } from '../tasks/login';
+import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
+
+import { CASES } from '../urls/navigation';
+
+describe('Cases', () => {
+  before(() => {
+    esArchiverLoad('timeline');
+  });
+
+  after(() => {
+    esArchiverUnload('timeline');
+  });
+
+  it('Creates a new case with timeline and opens the timeline', () => {
+    loginAndWaitForPageWithoutDateRange(CASES);
+    goToCreateNewCase();
+    createNewCase(case1);
+    backToCases();
+
+    cy.get(ALL_CASES_PAGE_TITLE).should('have.text', 'Cases Beta');
+    cy.get(ALL_CASES_OPEN_CASES_STATS).should('have.text', 'Open cases1');
+    cy.get(ALL_CASES_CLOSED_CASES_STATS).should('have.text', 'Closed cases0');
+    cy.get(ALL_CASES_OPEN_CASES_COUNT).should('have.text', 'Open cases (1)');
+    cy.get(ALL_CASES_CLOSED_CASES_COUNT).should('have.text', 'Closed cases (0)');
+    cy.get(ALL_CASES_REPORTERS_COUNT).should('have.text', 'Reporter1');
+    cy.get(ALL_CASES_TAGS_COUNT).should('have.text', 'Tags2');
+    cy.get(ALL_CASES_NAME).should('have.text', case1.name);
+    cy.get(ALL_CASES_REPORTER).should('have.text', case1.reporter);
+    case1.tags.forEach((tag, index) => {
+      cy.get(ALL_CASES_TAGS(index)).should('have.text', tag);
+    });
+    cy.get(ALL_CASES_COMMENTS_COUNT).should('have.text', '0');
+    cy.get(ALL_CASES_OPENED_ON).should('include.text', 'ago');
+    cy.get(ALL_CASES_SERVICE_NOW_INCIDENT).should('have.text', 'Not pushed');
+    cy.get(ALL_CASES_DELETE_ACTION).should('exist');
+    cy.get(ALL_CASES_CLOSE_ACTION).should('exist');
+
+    goToCaseDetails();
+
+    const expectedTags = case1.tags.join('');
+    cy.get(CASE_DETAILS_PAGE_TITLE).should('have.text', case1.name);
+    cy.get(CASE_DETAILS_STATUS).should('have.text', 'open');
+    cy.get(CASE_DETAILS_USER_ACTION)
+      .eq(USER)
+      .should('have.text', case1.reporter);
+    cy.get(CASE_DETAILS_USER_ACTION)
+      .eq(ACTION)
+      .should('have.text', 'added description');
+    cy.get(CASE_DETAILS_DESCRIPTION).should(
+      'have.text',
+      `${case1.description} ${case1.timeline.title}`
```

Comment:
you are assuming the timeline exists. you'll have to start this test by creating the timeline
<img width="1400" alt="Screen Shot 2020-04-20 at 12 17 49 PM" src="https://user-images.githubusercontent.com/6935300/79785354-f7de4880-8300-11ea-890d-277584983ac0.png">